setuptools/build_meta.py

```diff
@@ -46,6 +46,8 @@
            'prepare_metadata_for_build_wheel',
            'build_wheel',
            'build_sdist',
+           'get_requires_for_build_editable',
```

Comment:
@abravalheri You forgot to add `'prepare_metadata_for_build_editable'` to the `__all__` list.

Reply:
Thank you very much @dalcinl. I am addressing this on https://github.com/pypa/setuptools/pull/3477.